src/client/theme-default/components/VPTeamMembersItem.vue

```diff
@@ -180,6 +178,13 @@ defineProps<{
   margin: 0 auto;
 }
 
+.desc ::v-deep(a) {
```

Comment:
`::v-deep` is deprecated. Use `:deep` instead.

Reply:
I'm sorry, it's been corrected